src/contexts/UserContext.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, createContext } from "react";
+
+const UserContext = createContext();
+
+const UserProvider = (props) => {
+  const [user, setUser] = useState({
```

Comment:
This should come from the API/database so that its dynamic in the case of an added user